flake.nix

```
@@ -89,6 +89,10 @@
                   })
                   # The rust-analyzer language server implementation.
                   rust-analyzer
+                  # GCC includes a linker; needed for building `ruff`
```

```suggestion

                  # GCC includes a linker; needed for building `ruff`
```

I've been trying to separate distinct sections with a space :)